src/generic/json_structural_indexer.h

```diff
@@ -61,7 +61,13 @@ class json_structural_indexer {
   static error_code index(const uint8_t *buf, size_t len, parser &parser, bool streaming) noexcept;
 
 private:
-  really_inline json_structural_indexer(uint32_t *structural_indexes) : indexer{structural_indexes} {}
+  really_inline json_structural_indexer(uint32_t *structural_indexes)
```

Comment:
Please remove these changes to the constructor. It will handled in a distinct PR.

Reply:
done